mcux/scripts/pinctrl/README.md

```diff
@@ -0,0 +1,119 @@
+# MCUXpresso Pin Control Generation Scripts
+
+These scripts aid in the generation of pin control data NXP platforms. The
+scripts currently support most iMX.RT processors, and some LPC and Kinetis
+processors
+
+| Script                     | Functionality                               |
+| -------------------------- | ------------------------------------------- |
+| `gen_board_pinctrl.py`     | Generates pin control dtsi files for boards |
+| `gen_soc_headers.py`       | Generates pin control dtsi files defining SOC pinmux settings |
+
+## Requirements
+
+All scripts here utilize files generated by with [MCUXpresso Config
+tools v13](https://www.nxp.com/design/software/development-software/mcuxpresso-software-and-tools-/mcuxpresso-config-tools-pins-clocks-peripherals:MCUXpresso-Config-Tools),
+and version 13 data packs downloaded from
+[MCUXpresso SDK builder](https://mcuxpresso.nxp.com/en/select_config_tools_data)
+No other version of MCUXpresso Config tools is supported.
+MCUXpresso config tools data sources must be provided to this tool, see below
+for documentation on how to download data sources.
+
+
+## Supported SOCs
+
+The following SOC lines are supported:
+- i.MX RT 10xx series
+- i.MX RT 11xx series
+- Kinetis K8x, K6x, K2x series
+- LPC 55sxx series
+
```

Comment:
@DerekSnell  FYI- this is the list of SOCs I expect we should maintain support for with this script, please let me know if you think there are additional SOCs we should add (or any we should remove) from the official support list

Reply:
RT5xx & RT6xx?

Reply:
> RT5xx & RT6xx?

Thanks, added